content/concepts/plugins.md

```diff
@@ -122,3 +122,7 @@ Available plugins include:
 ## How to Write a Plugin
 
 For information on writing a new plugin, refer to the development documentation on [writing plugins]({{< ref "plugin_writing.md" >}}).
+
+## Collection Timeout
+
+CRS used to define `SecCollectionTimeout` in `crs-setup.conf` before but removed this setting with the introduction of plugins for CRS v4. That's because CRS itself does not work with collections anymore. However, for plugins, there may be a need to work with collections and maybe also to set a custom `SecCollectionTimeout` outside of the default 3600 seconds defined by the ModSecurity engine. If that's the case, the plugin should either set it in its configuration or indicate the desired value in the plugin documentation.
```

Comment:
Nit: to increase the readability making the text more action-oriented, I would probably write it the other way around: first explaining what has to be done about collection timeouts and only afterward providing historical context. Something like:
```suggestion
If plugins need to work with collections and set a custom `SecCollectionTimeout` outside of the default 3600 seconds defined by the ModSecurity engine, the plugin should either set it in its configuration or indicate the desired value in the plugin documentation. CRS used to define `SecCollectionTimeout` in `crs-setup.conf` before but removed this setting with the introduction of plugins for CRS v4. That's because CRS itself does not work with collections anymore. 
```
That being said, either way, looks good to me.